### PR TITLE
Karbranch

### PR DESCRIPTION
**Pull request by karsarthemad**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,4 +4,4 @@ Idle game by Kars and Zimmy
 
 This is a change to commit
 
-Push commit test #1
\ No newline at end of file
+Kars Branch?
```
Comment (jonathanmaxey) [inline README.md:7]: I think this is a kars branch? shoudl we add it

**Review by jonathanmaxey — CHANGES_REQUESTED** [review]
Make requested changes

File: README.md (hunk 1 above)
Comment (karsarthemad) [reply README.md:7]: no

**Review by jonathanmaxey — APPROVED** [review]
I guess lets merge

**Pull request merged**